shared/native/notification-listeners.native.js

```diff
@@ -1,37 +1,27 @@
 // @flow
-import shared from './notification-listeners.shared'
+import sharedNotificationActions from './notification-listeners.shared'
 import RNPN from 'react-native-push-notification'
 import * as RPCTypes from '../constants/types/rpc-gen'
+import engine from '../engine'
 
-import type {Dispatch} from '../constants/types/flux'
-import type {IncomingCallMapType} from '../constants/types/rpc-gen'
+// TODO: DESKTOP-6662 - Move notification listeners to their own actions
+export default (): void => {
+  sharedNotificationActions()
 
-// TODO(mm) Move these to their own actions
-export default function(dispatch: Dispatch, getState: () => Object, notify: any): IncomingCallMapType {
-  const fromShared: IncomingCallMapType = shared(dispatch, getState, notify)
-  const handlers: IncomingCallMapType = {
-    'keybase.1.NotifyBadges.badgeState': ({badgeState}) => {
-      const sharedBadgeState = fromShared['keybase.1.NotifyBadges.badgeState']
-      if (sharedBadgeState) {
-        sharedBadgeState({badgeState})
-      }
+  engine().setIncomingActionCreators('keybase.1.NotifyBadges.badgeState', ({badgeState}) => {
+    // const sharedBadgeState = fromShared['keybase.1.NotifyBadges.badgeState']
+    // if (sharedBadgeState) {
+    //   sharedBadgeState({badgeState})
+    // }
```

Comment:
I don't know how to replicate this with the move to `setIncomingActionCreators`.